### DISCUSSION TITLE
Restore unsaved scene

### DISCUSSION THREAD

**Discussion by inbaryeh** [excalidraw/excalidraw#8749] [General]
I accidentally replaced a scene with an older one; although I saved it to the disk beforehand, I can't find the file. Is there a way to restore it?

**Comment by dwelle** [comment]
Hi. If you don't have the local file, then no, since the data is stored in the browser only. If you haven't yet closed or reloaded the tab, you can try to undo using `ctrl/cmd-Z` (or from the UI, bottom left).

**Reply by inbaryeh** [reply]
Thanks. Unfortunately, I closed the the origin tab.
